sockeye/training.py

```diff
@@ -523,6 +523,8 @@ def fit(self,
         speedometer = Speedometer(frequency=C.MEASURE_SPEED_EVERY, auto_reset=False)
         tic = time.time()
 
+        converged = False
```

Comment:
not really important, but would it make sense to move this boolean into `TrainState` just for clarity?
In that case, this function could even return the TrainState object, which holds all kinds of information from training (best metric, updates, epochs etc.)

Reply:
Makes sense, actually this may simplify calling training from some code.